docs/6.x/changelog.md

```diff
@@ -340,6 +348,18 @@ LTS starts with `3.51.0` with minor backwards compatible changes added over time
 * Skip missing mount points when checking filesystem usage ([#786](https://github.com/gravitational/gravity/pull/786)).
 
 
+### 5.5.23 LTS (October 11th, 2019)
+
+#### Improvements
+
+* Only capture world-readable details in 'gravity report' ([#787](https://github.com/gravitational/gravity/pull/787)).
```

Comment:
Nit: Use backticks (\`) instead of apostrophes so the command is highlighted in rendered markdown? Same for `tele build` below.